res/css/views/rooms/_EventTile.scss

```diff
@@ -41,6 +41,11 @@ limitations under the License.
 
 .mx_EventTile_continuation {
     padding-top: 0px ! important;
+
+    &.mx_EventTile_isEditing {
+        padding-top: 5px ! important;
```

Comment:
At least for new rules, we should probably write `!important` without the space...